test.js

```diff
@@ -8,3 +8,19 @@ test('async', async t => {
 test('sync', t => {
 	t.is(runAppleScriptSync('return "unicorn"'), 'unicorn');
 });
+
+test('non human readable output async', async t => {
```

Comment:
```suggestion
test('non-human readable output - async', async t => {
```

Reply:
Fixed in [8ae5a3d](https://github.com/sindresorhus/run-applescript/pull/14/commits/8ae5a3dceab125fbda2be9ef95a515efbdfc40bc).